external/storm-metrics/pom.xml

```diff
@@ -48,6 +48,13 @@
       </exclusions>
     </dependency>
     <dependency>
+      <groupId>org.fusesource</groupId>
+      <artifactId>sigar</artifactId>
+      <version>1.6.4</version>
```

Comment:
We should also make this version a property or set in the parent pom, because there are multiple packages tied to this same version.  And we should remove the comment on line 39, as it is not needed.